app/views/layouts/application.html.erb

```diff
@@ -32,6 +32,9 @@
     <%= javascript_include_tag "application" %>
     <%= csrf_meta_tags %>
 
+    <script src="https://maps.googleapis.com/maps/api/js?libraries=places&language=en&key=AIzaSyDWgc7p4WWFsO3y0MTe50vF4l4NUPcPuwE"></script>
```

Comment:
Do we need this for every page? I think perhaps it should be in only more specific templates.